### PR TITLE
Data service

### PR DESCRIPTION
**Pull request by maclewan**
Data service implementation

### CODE REVIEW

File: app/src/main/java/com/piwniczna/mojakancelaria/DB/DataService.kt [hunk 1]
```diff
@@ -29,32 +29,66 @@ class DataService(context: Context) {
...
+
+    fun getNotPayedObligations(clientId: Int) : ArrayList<ObligationEntity> {
+        return ArrayList(db.getNotPayedObligations(clientId))
+    }
+
+    fun deleteObligation(obligation: ObligationEntity) {
+        return db.deleteObligation(obligation)
+    }
+
+
```
Comment (wiktoriasalamon) [inline app/src/main/java/com/piwniczna/mojakancelaria/DB/DataService.kt:58]: where is update?

File: app/src/main/java/com/piwniczna/mojakancelaria/activities/clients/ClientsFragment.kt [hunk 2]
```diff
@@ -132,4 +139,27 @@ class ClientsFragment : Fragment() {
         )?.commit()
     }
 
+    fun test() {
```
Comment (wiktoriasalamon) [inline app/src/main/java/com/piwniczna/mojakancelaria/activities/clients/ClientsFragment.kt]: Can you delete this?

Comment (maclewan) [reply app/src/main/java/com/piwniczna/mojakancelaria/activities/clients/ClientsFragment.kt]: yup

File: app/src/main/java/com/piwniczna/mojakancelaria/activities/clients/ClientsFragment.kt [hunk 3]
```diff
@@ -14,10 +14,15 @@ import android.widget.ListView
 import androidx.fragment.app.Fragment
 import com.piwniczna.mojakancelaria.DB.DataService
 import com.piwniczna.mojakancelaria.Models.ClientEntity
+import com.piwniczna.mojakancelaria.Models.ObligationEntity
```
Comment (wiktoriasalamon) [inline app/src/main/java/com/piwniczna/mojakancelaria/activities/clients/ClientsFragment.kt]:
```suggestion

```

File: app/src/main/java/com/piwniczna/mojakancelaria/activities/clients/ClientsFragment.kt [hunk 4]
```diff
@@ -14,10 +14,15 @@ import android.widget.ListView
...
+import com.piwniczna.mojakancelaria.Models.ObligationType
```
Comment (wiktoriasalamon) [inline app/src/main/java/com/piwniczna/mojakancelaria/activities/clients/ClientsFragment.kt]:
```suggestion

```

File: app/src/main/java/com/piwniczna/mojakancelaria/activities/clients/ClientsFragment.kt [hunk 5]
```diff
@@ -14,10 +14,15 @@ import android.widget.ListView
...
+import com.piwniczna.mojakancelaria.Models.PaymentEntity
```
Comment (wiktoriasalamon) [inline app/src/main/java/com/piwniczna/mojakancelaria/activities/clients/ClientsFragment.kt]:
```suggestion

```

File: app/src/main/java/com/piwniczna/mojakancelaria/activities/clients/ClientsFragment.kt [hunk 6]
```diff
@@ -14,10 +14,15 @@ import android.widget.ListView
...
 import com.piwniczna.mojakancelaria.R
 import com.piwniczna.mojakancelaria.utils.SpannedText
 import com.piwniczna.mojakancelaria.activities.add_client.AddClientFragment
 import com.piwniczna.mojakancelaria.activities.client_details.ClientDetailsFragment
+import java.math.BigDecimal
```
Comment (wiktoriasalamon) [inline app/src/main/java/com/piwniczna/mojakancelaria/activities/clients/ClientsFragment.kt]:
```suggestion

```

File: app/src/main/java/com/piwniczna/mojakancelaria/activities/clients/ClientsFragment.kt [hunk 7]
```diff
@@ -14,10 +14,15 @@ import android.widget.ListView
...
+import java.time.LocalDate
```
Comment (wiktoriasalamon) [inline app/src/main/java/com/piwniczna/mojakancelaria/activities/clients/ClientsFragment.kt]:
```suggestion

```

**Pull request merged**